README.md

```diff
@@ -21,3 +21,15 @@ $ make
 
 [nasm]: https://www.nasm.us/
 [qemu]: https://www.qemu.org/
+
+## Building on MacOS
+
+1. `brew install nasm`
+2. `brew install qemu`
+
+### Running
+
+1. `make`
```

Comment:
Completely redundant information. We already have that in "Quick Start" section.